src/main/java/io/github/essencepowered/essence/commands/kit/KitCommand.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * This file is part of Essence, licensed under the MIT License (MIT). See the LICENSE.txt file
+ * at the root of this project for more details.
+ */
+package io.github.essencepowered.essence.commands.kit;
+
+import com.google.common.collect.Lists;
+import com.google.inject.Inject;
+import io.github.essencepowered.essence.api.PluginModule;
+import io.github.essencepowered.essence.config.KitsConfig;
+import io.github.essencepowered.essence.internal.CommandBase;
+import io.github.essencepowered.essence.internal.KitInventory;
+import io.github.essencepowered.essence.internal.annotations.Modules;
+import io.github.essencepowered.essence.internal.annotations.Permissions;
+import io.github.essencepowered.essence.internal.annotations.RegisterCommand;
+import io.github.essencepowered.essence.internal.permissions.SuggestedLevel;
+import org.spongepowered.api.command.CommandResult;
+import org.spongepowered.api.command.CommandSource;
+import org.spongepowered.api.command.args.CommandContext;
+import org.spongepowered.api.command.args.GenericArguments;
+import org.spongepowered.api.command.spec.CommandSpec;
+import org.spongepowered.api.entity.living.player.Player;
+import org.spongepowered.api.item.inventory.Inventory;
+import org.spongepowered.api.text.Text;
+
+import java.util.List;
+
+/**
+ * Allows a user to warp to the specified warp.
+ *
+ * Command Usage: /kit Permission: essence.kit.base
+ */
+@Permissions(suggestedLevel = SuggestedLevel.ADMIN)
+@Modules(PluginModule.KITS)
+@RegisterCommand("world")
+public class KitCommand extends CommandBase<CommandSource> {
+
+    private final String kit = "kit";
+
+    @Inject private KitsConfig kitConfig;
+
+    @Override
+    public CommandSpec createSpec() {
+        return CommandSpec.builder().executor(this).children(this.createChildCommands())
+                .arguments(GenericArguments.onlyOne(GenericArguments.string(Text.of(kit)))).build();
```

Comment:
Could probably make this an arugment parser instead. That way, we don't have to duplicate logic if we need to select from a set of kits.

I did this with Warps - https://github.com/NucleusPowered/Nucleus/blob/master/src/main/java/io/github/essencepowered/essence/argumentparsers/WarpParser.java